### ISSUE TITLE
XBLA games no longer work by folder drag and drop

### ISSUE THREAD

**Issue by Rex000** [xenia-canary/xenia-canary#260]
### Validation

- [X] I've read the [FAQ](https://github.com/xenia-project/xenia/wiki/FAQ).
- [X] The Xenia build used is from the master branch (not MLBS/AlexVS/Canary/pull requests, etc.)
- [X] This issue isn't for tech support (help with Xenia).
- [X] If this issue occurs in a specific game, I've done analysis to locate the faulty subsystem of the emulator and a potential reason in it.
- [X] I've checked if this issue hasn't already been reported.
- [X] My device meets the minimum requirements: https://github.com/xenia-project/xenia/wiki/Quickstart#system-requirements
- [X] (If building) I have read the building doc: https://github.com/xenia-project/xenia/blob/master/docs/building.md

### Describe what's going wrong

XBLA games no longer work by folder drag and drop. It started happening since the following build

https://github.com/xenia-canary/xenia-canary/releases/tag/18daf4a

Now it gives this error upon drag and drop of game folder
![error0](https://github.com/xenia-canary/xenia-canary/assets/28770012/5f6ab893-d73f-43e0-98b9-fa42aa99b3b1)

### Describe what should happen

Should work

### If applicable, upload a logfile and link it here

[xenia.log](https://github.com/xenia-canary/xenia-canary/files/14425994/xenia.log)

**Comment by Ivan951236** [comment]
For me it works :/
Because of that, i think that your XBLA game rom is a bad dump, dump again

**Comment by Rex000** [comment]
> For me it works :/ Because of that, i think that your XBLA game rom is a bad dump, dump again

No it's not a bad dump. You never mentioned what version exactly you are using. If mine was a bad dump then it wouldn't have worked on ANY build of Xenia Canary. It works fine on builds before September.

**Comment by Gliniak** [comment]
Please provide log from such run (set log_level to 3 in config)

**Comment by Rex000** [comment]
Added the log

**Comment by Gliniak** [comment]
It looks like it crashed before even loading anything. 

I think you have to provide package or default.xex directly not whole folder. If it worked like that then it was probably not intentional.
However I will check how it works on master build and canary just on case

**Comment by Ivan951236** [comment]
> > For me it works :/ Because of that, i think that your XBLA game rom is a bad dump, dump again
> 
> No it's not a bad dump. You never mentioned what version exactly you are using. If mine was a bad dump then it wouldn't have worked on ANY build of Xenia Canary. It works fine on builds before September.

my xenia's (canary) version is this
![image](https://github.com/xenia-canary/xenia-canary/assets/101629412/751f09ab-b51c-4edc-be84-8009707bbaf6)

**Comment by Rex000** [comment]
> It looks like it crashed before even loading anything.
> 
> I think you have to provide package or default.xex directly not whole folder. If it worked like that then it was probably not intentional. However I will check how it works on master build and canary just on case

Yes, it already works if I navigate to the default.xex file using the Ctrl+O. Though it used to work by just folder drag and drop in Xenia builds up to August.

**Comment by Rex000** [comment]
> It looks like it crashed before even loading anything.
> 
> I think you have to provide package or default.xex directly not whole folder. If it worked like that then it was probably not intentional. However I will check how it works on master build and canary just on case

XBLA games are just one single file. And yes, it already works if I navigate to that extension-less file using Ctrl+O. Though it used to work by just folder drag and drop in canary builds up to August.

**Comment by Gliniak** [comment]
I checked in code. This is intended. 
It was removed due to changes in how these packages works, as these packages now are working a bit differently than on master build.
I do not plan to restore this functionality. 
It worked yes, but it had one huge assumption. If you had multiple packages in same directory it would select random one. This is misleading and might cause issues, so it's better to explicitly provide what you want to boot
